python/lsst/daf/butler/tests/_dummyRegistry.py

```diff
@@ -150,7 +153,8 @@ class DummyRegistry:
     def __init__(self):
         self._opaque = DummyOpaqueTableStorageManager()
         self.dimensions = DimensionUniverse()
-        self._datastoreBridges = DummyDatastoreRegistryBridgeManager(self._opaque, self.dimensions)
+        self._datastoreBridges = DummyDatastoreRegistryBridgeManager(
+            self._opaque, self.dimensions, sqlalchemy.BigInteger)
```

Comment:
Does this mean the tests are not using UUID?

Reply:
I think it it does not really matter, `DummyDatastoreRegistryBridgeManager` does not use that type at all. But it is needed there because base class constructor uses it. Looking again at base class I'm not sure now why I added that attribute to the base class, maybe it should be defined in subclasses instead, I'll look again.

Reply:
This is complicated. `datasetIdColumnType` is defined in `DatastoreRegistryBridgeManager` so I need to initialize with something. `datasetIdColumnType` needs to be in the base class because FileDatastore uses it to make its table schema. Normally (in regular Registry) this type is obtained from `DatasetRecordStorageManager` (obviously it needs to be consistent) but in case of DummyRegistry there is no datasets manager at all. I do not know if unit tests actually instantiate FileDatastore from a DummyRegistry (they probably do). If we want to run those tests with both integer and UUID IDs then we need to extend them and probably define DummyRegistryUUID class. This sort of repetition is already very ugly,m I had to parameterize bunch of tests in `test_simpleButler.py` and it has all chances to grow if we add new manager implementations.

Reply:
I wouldn't worry too much about it for now. I had forgotten where DummyRegistry was being used -- the standalone datastore test really doesn't care what the id is.